config/maintainers.json

```diff
@@ -22,10 +22,10 @@
     },
     {
       "github_username": "NobbZ",
-      "show_on_website": false,
+      "show_on_website": true,
       "alumnus": false,
-      "name": null,
-      "bio": null,
+      "name": "Norbert Melzer",
+      "bio": "Found out about erlang via exercism, got hooked up and after fixing some trivial bugs, I have been added to the team,",
```

Comment:
@iHiD do you have any sketches available for the maintainer bios and what they might look like on the site? Also how long should these be? (I was thinking a paragraph or two).